pkg/proxy/winkernel/proxier.go

```diff
@@ -477,6 +483,12 @@ func NewProxier(
 
 	glog.V(1).Infof("Hns Network loaded with info = %v", hnsNetwork)
 
+	sourceVip := os.Getenv("SOURCE_VIP")
+	hostMac := os.Getenv("HOST_MAC")
```

Comment:
even if this is OS specific, shouldn't KubeProxyConfiguration be used instead of environment variables,?